ash-window/src/lib.rs

```diff
@@ -125,22 +110,14 @@ pub fn enumerate_required_extensions(
     window_handle: &dyn HasRawWindowHandle,
```

Comment:
this could be switched to use the display handle instead, removing the requirement for a window - should help in particular with Androids `Suspend`/`Resume` limitations

Reply:
That's a nice suggestion! However, that means we should assume that whenever the user can (easily) get to a `HasRawWindowHandle`, they also have access to `HasRawDisplayHandle`. This is true in a "chronological" sense (you need a "display" to create a "window") and for Winit where both `Window` and `EventLoopWindowTarget` (and `EventLoop` which coerces into that) implement both traits.

It doesn't matter for `Android` since `winit::Window` is just an empty shell there that you can create well before receiving `Event::Resumed`.

Reply:
One thing to note is that @kchibisov simplified the names somewhat. `AndroiNdk` was generalized to `Android` [because it's only representing the Android platform and not specifically an "NDK" `NativeWindow` pointer (Java `Surface`)](https://github.com/rust-windowing/raw-window-handle/pull/93#discussion_r917000491).

The same was done for `Windows` where there's no separation between `Win32` and `WinRt` in the "display" handle. But as soon as Vulkan support for a `WinRt` handle starts trickling in ([efforts are silent/stale](https://github.com/KhronosGroup/Vulkan-Docs/issues/366)) `ash-window` needs to distinguish them on `enumerate_required_extensions()` as well.

Alas, I see no reason to preempt them now though; we can always update later when it is clear _how_. Just thought we should all be aware of this.